ash-window/src/lib.rs

```diff
@@ -118,6 +118,59 @@ pub unsafe fn create_surface(
     }
 }
 
+/// Extensions necessary for creating a surface on windows.
+#[allow(dead_code)]
+const WINDOWS_SURFACE_EXTENSIONS: &'static [*const c_char] = &[
+    khr::Surface::name().as_ptr(),
+    khr::Win32Surface::name().as_ptr(),
+];
+/// Extensions necessary for creating a surface on unix.
+/// Note that this is not equal to the return value of [`enumerate_required_extensions`] on Unix due to the multiple window types.
```

Comment:
How about elaborating this to return _all possible_ extensions on Unix, and explaining how `enumerate_required_extensions()` differs in that situation?

```suggestion
/// _All_ possible extensions for creating a surface on Unix.
///
/// Defer to [`enumerate_required_extensions()`] for a subset of this list applicable to the display server protocol specific to the given [`HasRawWindowHandle`].
```

Also add this to `TARGET_SURFACE_EXTENSIONS` since `UNIX_SURFACE_EXTENSIONS` is not public :disappointed: 

(If you intend these to be separate paragraphs, add an empty `///` in between. Otherwise the two sentences get concatenated into one paragraph.)